model_compression_toolkit/gptq/pytorch/gptq_training.py

```diff
@@ -70,6 +73,7 @@ def __init__(self,
             hessian_info_service: HessianInfoService to fetch info based on the hessian approximation of the float model.
         """
         def _get_total_grad_steps():
+            # TODO get it from the dataset
```

Comment:
It's not possible after this refactor? (probably not but I'm just making sure)

Reply:
Not sure if there was an issue here or I just didn't want to waste time on something that currently works as I was instructed :)  Most probably it's the last one